pkg/util/ephemeral/label_matchers.go

```diff
@@ -209,7 +209,9 @@ func matchersConfigString(matchers map[Source][]string) string {
 }
 
 // MarshalYAML implements yaml.Marshaler.
-func (c *LabelMatchers) MarshalYAML() (interface{}, error) {
+//
+//goland:noinspection GoMixedReceiverTypes // We need MarshalYAML to be on non-pointer as validation.Limits doesn't use pointer to this struct.
```

Comment:
`//goland:...` directive is always on last line. But I've removed this comment completely now, as it would need to be on all methods in order to keep Goland quiet.